hoomd/mpcd/CellList.cc

```diff
@@ -352,12 +363,17 @@ void mpcd::CellList::buildCellList()
     const BoxDim& box = m_pdata->getBox();
     const uchar3 periodic = box.getPeriodic();
 
-    ArrayHandle<unsigned int> h_cell_list(m_cell_list,
-                                          access_location::host,
-                                          access_mode::overwrite);
     ArrayHandle<unsigned int> h_cell_np(m_cell_np, access_location::host, access_mode::overwrite);
+    ArrayHandle<double4> h_cell_vel(m_cell_vel, access_location::host, access_mode::overwrite);
+    ArrayHandle<double3> h_cell_energy(m_cell_energy,
+                                       access_location::host,
+                                       access_mode::overwrite);
+
     // zero the cell counter
     m_cell_np.zeroFill();
+    m_cell_vel.zeroFill();
+    m_cell_energy.zeroFill();
```

Comment:
I'm not sure exactly how `zeroFill` is implemented, but there are two things:
* I would fill first before acquiring the handle, since it is going to modify the values contained in the array.
* These should be accessed `readwrite` if they are going to be added into. It was probably fine before because of the combination of overwrite + fill, but I would still do it this way to be safe.
```suggestion
    // zero the cell counter
    m_cell_np.zeroFill();
    m_cell_vel.zeroFill();
    m_cell_energy.zeroFill();

    ArrayHandle<unsigned int> h_cell_np(m_cell_np, access_location::host, access_mode::readwrite);
    ArrayHandle<double4> h_cell_vel(m_cell_vel, access_location::host, access_mode::readwrite);
    ArrayHandle<double3> h_cell_energy(m_cell_energy,
                                       access_location::host,
                                       access_mode::readwrite);
```